README.md

```diff
@@ -46,9 +46,12 @@ pip install -r requirements.txt
 ```
 3. Download and extract the dataset
 ```bash
+mkdir datasets
 cd datasets
+mkdir mvtec3d
+cd mvtec3d
 wget https://www.mydrive.ch/shares/45920/dd1eb345346df066c63b5c95676b961b/download/428824485-1643285832/mvtec_3d_anomaly_detection.tar.xz
-tar -xvf mvtec_3d_anomaly_detection.tar.xz && mv mvtec_3d_anomaly_detection mvtec3d
+tar -xvf mvtec_3d_anomaly_detection.tar.xz 
```

Comment:
I don't think there is a need for these changes, the original command should cover this as the downloaded tar is extracted into the `mvtec_3d_anomaly_detection` dir and the `&& mv mvtec_3d_anomaly_detection mvtec3d` part renames it to `mvtec3d`

Reply:
As the extraction of mvtec_3d_anomaly_detection.tar.gz would not produce the folder mvtec_3d_anomaly_detection, so mvtec_3d_anomaly_detection folder should be made before extraction.